scripts/deploy-gh.sh

```diff
@@ -11,8 +11,6 @@ git merge master --no-edit
 
 # build
 npm run presite
```

Comment:
I try to track but can't find where  we run `build:examples`? (Maybe I'm just tired and overlook them.)